jena-core/src/main/java/org/apache/jena/vocabulary/SO.java

```diff
@@ -0,0 +1,2366 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.jena.vocabulary;
+
+import org.apache.jena.rdf.model.Model;
+import org.apache.jena.rdf.model.ModelFactory;
+import org.apache.jena.rdf.model.Property;
+import org.apache.jena.rdf.model.Resource;
+
+public class SO {
+
+/**
+ *  Schema.org is a collaborative, community activity with a mission to create, maintain, and promote schemas for structured data on the Internet, on web pages, in email messages, and beyond.
+ *  <p>
+ *	See <a href="https://schema.org/">Schema.org</a>.
+ *  <p>
+ *  <a href="https://schema.org/">Base URI and namepace</a>.
+ */
+    private static final Model m = ModelFactory.createDefaultModel();
+    public static final String NS = "https://schema.org/";
```

Comment:
FYI:
schema.org might be "http://" as well. There is no right answer here - it's at the whim of the JSON-LD context. So I asked and got a response that "https" is slightly better. We'll see.

Reply:
I changed the name.